qiskit/visualization/latex.py

```diff
@@ -157,30 +157,19 @@ def latex(self):
 
         self._initialize_latex_array()
         self._build_latex_array()
-        header_1 = r"""% \documentclass[preview]{standalone}
-% If the image is too large to fit on this documentclass use
-\documentclass[draft]{beamer}
-"""
-        beamer_line = "\\usepackage[size=custom,height=%d,width=%d,scale=%.1f]{beamerposter}\n"
```

Comment:
Currently the scaling of the latex output is done in this line. Since this is being removed, I guess the scaling would not be possible now, i.e, qc.draw('latex', scale=0.5) and qc.draw('latex', scale=0.1) would return the same output. Could you think of a way around to have scaling while still using standalone?

Reply:
Got it, I'll try to think of a way to include scaling.